charts/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,319 @@
+the exporter is logging exporter.
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: my-collector
+spec:
+  mode: deployment # This configuration is omittable.
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+    processors:
+
+    exporters:
+      logging:
+
+    service:
+      pipelines:
+        traces:
+          receivers: [jaeger]
+          processors: []
+          exporters: [logging]
+EOF
+```
+
+### DaemonSet Mode
+
+DaemonSet should satisfy your needs if you want the Collector run as an agent in your Kubernetes nodes.
+In this case, every Kubernetes node will have its own Collector copy which would monitor the pods in it.
+
+The following example configuration deploys the Collector as DaemonSet resource. The receiver is Jaeger receiver and
+the exporter is logging exporter.
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: my-collector
+spec:
+  mode: daemonset
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+    processors:
+
+    exporters:
+      logging:
+        loglevel: debug
+
+    service:
+      pipelines:
+        traces:
+          receivers: [jaeger]
+          processors: []
+          exporters: [logging]
+EOF
+```
+
+### StatefulSet Mode
+There are basically three main advantages to deploy the Collector as the StatefulSet:
+- Predictable names of the Collector instance will be expected \
+  If you use above two approaches to deploy the Collector, the pod name of your Collector instance will be unique (its name plus random sequence).
+  However, each Pod in a StatefulSet derives its hostname from the name of the StatefulSet and the ordinal of the Pod (my-col-0, my-col-1, my-col-2, etc.).
+- Rescheduling will be arranged when a Collector replica fails \
+  If a Collector pod fails in the StatefulSet, Kubernetes will attempt to reschedule a new pod with the same name to the same node. Kubernetes will also attempt
+  to attach the same sticky identity (e.g., volumes) to the new pod.
+- The load balancer could be configured (under construction) \
+  The load balancer will use a HTTP server to expose the scrape targets to a specific endpoint URL, which will be used by the Prometheus receiver to scrape metrics
+  data. Additionally, the load balancer will use that discovery information to evenly delegate scraping jobs to the collector instances inside a StatefulSet based on
+  a replica's current workload.
+
+The following example configuration deploys the Collector as StatefulSet resource with three replicas. The receiver
+is Jaeger receiver and the exporter is logging exporter.
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: my-collector
+spec:
+  mode: statefulset
+  volumeMounts:
```

Comment:
The collector isn't using this volume in the configuration. This isn't needed, is it?

Reply:
Yes, this configuration is neglectable since the Collector doesn't use it at this time.